src/parse/load_from.jl

```diff
@@ -0,0 +1,175 @@
+"""
```

Comment:
**Organization** This file is functions related to loading data types from a structure like a DataFrame.